src/content/deployment/subctl.en.md

```diff
@@ -0,0 +1,184 @@
+                                         as gateway.
+
+    --subm-debug                         Enable Submariner debugging (verbose logging)
+
+    --disable-cvo                        disable OpenShift's cluster version operator if necessary, without prompting
+                                         currently, lighthouse service modifies the cluster dns operator in OpenShift
+                                         and we need to disable CVO, this setting disables the prompt which warns the
+                                         administrator.
+
+##### join flags (globalnet)
+
+    --globalnet-cluster-size uint        Cluster size for GlobalCIDR allocated to this cluster (amount of global IPs).
+
+    --globalnet-cidr string              GlobalCIDR to be allocated to the cluster, this setting is exclusive with
+                                         --globalnet-cluster-size and configures a specific globalnet CIDR for this
+                                         cluster.
+
+##### join flags (IPSec)
+
+    --disable-nat                        Disable NAT for IPsec.
+
+    --ikeport <value>                        IPsec IKE port (default 500)
+
+    --ipsec-debug                        Enable IPsec debugging (verbose logging)
+
+    --nattport int                       IPsec NATT port (default 4500)
+
+##### join flags (images and repositories)
+
+    --repository string                  image repository, the repository from where the various submariner images
+                                         will be sourced. (default "quay.io/submariner")
+
+    --version string                     image version
+
+    -o, --operator-image string          the operator image you wish to use
+                                         (default "quay.io/submariner/submariner-operator:$version")
+
+    --service-discovery-repo string      Service Discovery Image repository (default "quay.io/submariner")
+
+    --service-discovery-version string   Service Discovery Image version
+
+#### info
+
+`subctl info broker-info.subm [flags]`
+
+The `info` command prints basic information about the network discovery on the cluster. 
+
+##### info flags
+
+    -h, --help             help for info
+
+    --kubeconfig string    absolute path(s) to the kubeconfig file(s) (default "$HOME/.kube/config")
+
+    --kubecontext string   kubeconfig context to use
+
+#### verify-connectivity
+
+`subctl verify-connectivity <kubeConfig1> <kubeConfig2> [flags]`
+
+The `verify-connectivity` command verifies dataplane connectivity between two clusters. The
+`kubeConfig1` file will be `ClusterA` in the reports, while `kubeConfig2` will be `ClusterB` in the
+reports. The `--verbose` flag is recommended to see what's happening during the tests.
+
+Dataplane connectivity is verified in multiple ways: between pods and services, between pods and pods,
+and between gateway and non-gateway node combinations.
```

Comment:
Shall we use bullets? For example

* Pods to Pods
* Pods to Services
where Pods are scheduled on Gateway and non-Gateway nodes